package.json

```diff
@@ -38,13 +36,31 @@
     "socket.io-client": "^0.9.16"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "bunyan": "~0.22.3",
     "chai": "~1.9.1",
     "cookie-signature": "^1.0.5",
+    "eslint": "^6.6.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-config-standard-jsx": "^8.1.0",
+    "eslint-config-standard-react": "^9.2.0",
+    "eslint-plugin-chai-expect": "^2.1.0",
+    "eslint-plugin-chai-friendly": "^0.6.0",
+    "eslint-plugin-import": "^2.21.2",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-mocha": "^7.0.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-react": "^7.20.0",
+    "eslint-plugin-standard": "^4.0.1",
+    "mocha": "^4.0.1",
+    "prettier": "^2.0.5",
+    "prettier-eslint-cli": "^5.0.0",
     "sandboxed-module": "~0.3.0",
     "sinon": "^2.4.1",
-    "mocha": "^4.0.1",
-    "uid-safe": "^1.0.1",
-    "timekeeper": "0.0.4"
+    "timekeeper": "0.0.4",
+    "uid-safe": "^1.0.1"
```

Comment:
Can we rebase this PR onto the dependencies upgrade -- there is a conflict with `uid-safe`.